Create 2020-28-08-Open-Science-Saves-Lives.md (#102)
OpenScienceMOOC/site: _posts/2020-28-08-Open-Science-Saves-Lives.md

The article is available here: https://www.biorxiv.org/content/10.1101/2020.08.13.249847v1 

> While we may not all be working in medical research, we believe that our findings and recommendations extend to all fields of research and thereby invite you to co-sign this article to show that Open Science principles are supported by a large population of scientists. Signatures will be added to the paper as supplementary material

Reviewer: ```suggestion
While we may not all be working in medical research, we believe that our findings and recommendations extend to all fields of research and thereby invite you to co-sign this article to show that Open Science principles are supported by a large population of scientists. Signatures will be added to the paper as supplementary material.
```

> 

Reviewer: ```suggestion
(Note that co-signing means you agree with the problems and/or solution presented in the paper, you will not become a co-author of the article)
```

The process of co-signing this article will go as follow:
You will have until August 31 at midnight (PDT) to co-sign.
You have to provide an institutional email address that we will manually verify before sending you a verification email that will require action on your part.

> (We welcome feedback on the preprint and may make some minor changes to the text prior to submission.)

Reviewer: Why only minor ? What if there is a very critical review that would need major changes?

Author: We can't ask people to sign if the manuscript changes too much

> Join the 300+ co-signatories of the manuscript now: https://forms.gle/vagxi5mrQLi5hHpx5 

Reviewer: ```suggestion
We hope that the OSMooc community, who has always been very responsive and supportive of this kind of approach, will join our 300+ co-signatories. Please co-sign here:: https://forms.gle/vagxi5mrQLi5hHpx5 
```
